addons/hr_expense/models/hr_expense.py

```diff
@@ -1255,6 +1255,7 @@ def action_register_payment(self):
                 'active_model': 'account.move',
                 'active_ids': self.account_move_id.ids,
                 'default_partner_bank_id': self.employee_id.sudo().bank_account_id.id,
+                'default_group_payment': True,
```

Comment:
Shouldn't this be a `default=True` attribute on the field declaration directly?
Or maybe by adjusting the compute function?